view/administration/contacts/editContact/editContact.php

```diff
@@ -43,27 +43,135 @@
                         <label for="pTitle">Position Title:</label>
-                            <input type="text" class="form-control" id="pTitle">
+                            <input v-model="contact.title" type="text" class="form-control" id="pTitle">
                     </div>
                 </div>
                 <div class="col-md-3">
                     <div class="form-group"> 
                         <label class="required" for="pNumber">Phone Number</label>
-                            <input type="text" class="form-control" id="pNumber" required>
+                            <input v-model="contact.phoneNumber" type="text" class="form-control" id="pNumber" required>
                     </div>
                 </div>
             </div>
             <!-- next row -->
             <div class="row">
                 <div class="col-md-3">
-                    <button type="button" class="btn btn-success">Edit</button>
+                    <button @click="editContact" type="button" class="btn btn-success">Edit</button>
                 </div>
             </div>
         </div>
      </div>
+            <!-- Error Messages -->
+    <div v-for="(error, index) in errors" 
+         :key="index" class="alert alert-danger alert-dismissible fade show message-box" 
+         >
+        <button type="button" class="close" @click="clearError(index)">&times;</button>
+        {{error}}
+    </div>
+
+    <!-- Success Message -->
+    <div v-if="successMessage != null" 
+         class="alert alert-success alert-dismissible fade show message-box" 
+         >
+        <button type="button" class="close" @click="clearSuccessMessage">&times;</button>
+        {{successMessage}}
+    </div>
 </div>
 
 <link rel="stylesheet" href="../view/administration/contacts/contacts.css">
+<style scoped>
+.message-box {
+    position: fixed;
+    bottom: 0;
+    right: 5px;
+    width: 300px;
+}
+</style>
+<script>
+    new Vue ({
+        el: "#editContactApp",
+        data: {
+            id: null,
+            contact: {
+                firstName: null,
+                lastName: null,
+                email: null,
+                title: null,
+                phoneNumber: null               
+            },
+            errors: [],
+            successMessage: null           
+        },
+        created() {
+                this.getId();
+                this.getContact();
+        },
+        methods: {
+            getId(){
+                let uri = window.location.href.split('?');
+                if(uri.length === 2) {
+                  let vars = uri[1].split('&');
+                  let getVars = {};
+                  let tmp = '';
+                  vars.forEach(function(v) {
+                    tmp = v.split('=');
+                    if(tmp.length === 2)
+                      getVars[tmp[0]] = tmp[1];
+                  });
+                  this.id = getVars['id'];
+                }
+                },
```

Comment:
Fix indentation. My suggestion is to select lines 123 - 171 and do a `SHIFT` `TAB`